cluster-autoscaler/cloudprovider/packet/packet_manager_rest.go

```diff
@@ -372,18 +445,29 @@ func (mgr *packetManagerRest) deleteNodes(nodegroup string, nodes []NodeRef, upd
 				klog.Infof("nodegroup match %s %s", d.Hostname, n.Name)
 				if d.Hostname == n.Name {
 					klog.V(1).Infof("Matching Packet Device %s - %s", d.Hostname, d.ID)
-					req, _ := http.NewRequest("DELETE", mgr.baseURL+"/devices/"+d.ID, bytes.NewBuffer([]byte("")))
+
+					req, err := http.NewRequest("DELETE", mgr.baseURL+"/devices/"+d.ID, bytes.NewBuffer([]byte("")))
+					if err != nil {
+						return fmt.Errorf("failed to delete device: %w", err)
```

Comment:
Would it make sense to return here which device could not be deleted?
```suggestion
						return fmt.Errorf("failed to delete device %q: %w", d.ID, err)
```